src/pyclaw/fileio/ascii.py

```diff
@@ -48,6 +48,7 @@ def write(solution, frame, path, file_prefix='fort', write_aux=False,
         f.write("%5i                  nstates\n" % len(solution.states))
         f.write("%5i                  num_aux\n" % solution.num_aux)
         f.write("%5i                  num_dim\n" % solution.domain.num_dim)
+        f.write("%5i                  num_ghost\n" % 0)
```

Comment:
Do you also want to write `file_format` for consistency with the Fortran version?

Reply:
Okay, I've added that.